internal/system/agent/const.go

```diff
@@ -0,0 +1,6 @@
+package agent
```

Comment:
File should be removed.  It contains constants that are not referenced from anywhere.